src/screens/Customers/CustomerMain.tsx

```diff
@@ -100,6 +108,9 @@ function CustomerMain(props: RouteComponentProps & UserProps) {
           </FormControl>
         </div>
       </div>
+      <Fab href="/customers/create" color='primary' aria-label='add customer' className={classes.fab} size='medium'>
+        <AddIcon fontSize="large"/>
+      </Fab>
```

Comment:
We've been using `Link` components. Unless there's a good reason to switch off, let's keep this the standard. You can read about the difference [here](https://stackoverflow.com/questions/43087007/react-link-vs-a-tag-and-arrow-function). There are some examples of it being used throughout the codebase (like in this file).

A practical reason we're doing this is because non-Links cause refreshes (from the link) which can interrupt in-progress http requests (which we have to account for due to offline func since http requests can end up firing at any point the user is navigating the app)

Reply:
Got it. This also somehow fixed the issue where it was turning blue, so now it has the proper animations on click! 

Also good catch on the chevron poking over the button! I fixed it by moving the FAB below the customer cards.